mergify_engine/actions/merge_base.py

```diff
@@ -607,8 +611,9 @@ def get_queue_summary(self, ctxt: context.Context, q: queue.Queue) -> str:
         priorities_configured = False
 
         summary = "\n\nThe following pull requests are queued:"
+        pulls_priorities = {p: (await q.get_config(p))["priority"] for p in pulls}
```

Comment:
something like this should be better to do multiple calls at once:
```suggestion
        async def _get_config(p):
             return p, (await q.get_config(p))["priority"]
        pulls_priorities = dict(await asyncio.gather(*(_get_config(p) for p in pulls)))
```